stellar-contract-env-common/src/raw_val.rs

```diff
@@ -61,7 +61,21 @@ impl AsMut<RawVal> for RawVal {
 // away, the same way `()` would, while remaining a separate type to allow
 // conversion to a more-structured error code at a higher level.
 #[derive(Debug)]
-pub struct ConversionError;
+pub struct ConversionError<T> {
+    pub f: RawVal,
+    pub t: PhantomData<T>,
```

Comment:
Could we give these fields names? I find short names are useful when the field name would repeat the type (e.g. if the type name was already From, a field name of from is not useful), or when the field is constrained to a small scope, neither of which is the case here.
```suggestion
    pub from: RawVal,
    pub to: PhantomData<T>,
```

Reply:
Fixed.